Automattic/jetpack
# Protect: Stop tracking generated CSS file

Previously, after running `npm run build` on `master`, I'd get something like this:

![screen shot 2016-08-08 at 7 24 54 pm](https://cloud.githubusercontent.com/assets/1126811/17500755/d125b3a4-5d9d-11e6-9c7f-04e831eb1b58.png)

It looks like we began tracking changes to this file in 39d5166f16f2506bd56f86f77e8b026495348b24. Since it's a generated file, I don't think we should track it.

To test:

- Checkout `update/gitignore-react-build` branch
- In terminal `npm run build` 
- Run `git status`. Verify nothing shows

cc @jeffgolenski @lezama for review
